BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -112,6 +113,12 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
         {
             // TODO: Add all common Macrovision directory path checks here
 
+            var matchers = new List<PathMatchSet>
```

Comment:
Please follow the way the other checks work as well. As it stands, the path matchers you added are _only_ ever run if nothing else returned. I don't think that was your intended outcome.